rr.c

```diff
@@ -351,7 +351,8 @@ ldns_rr_new_frm_str_internal(ldns_rr **newrr, const char *str,
 		case LDNS_RDF_TYPE_WKS        : /* it is the last rd field. */
 		case LDNS_RDF_TYPE_IPSECKEY   :
 		case LDNS_RDF_TYPE_AMTRELAY   :
-		case LDNS_RDF_TYPE_NSEC       :	if (r_cnt == r_max - 1) {
+		case LDNS_RDF_TYPE_NSEC       :	
+		case LDNS_RDF_TYPE_SVCBPARAMS : if (r_cnt == r_max - 1) {
```

Comment:
I only partially understand what delimiters is for, but:
* Is there a need to restrict it to just \n?
* It seems like LDNS_RDF_TYPE_SVCBPARAMS is always the last rd field, so is it necessary to include it in this list?

Reply:
I believe it is necessary.  When I remove this, it fails to parse any record with more than one parameter.  Just having "alpn" works, but something like "alpn" and "port" fails.  So I believe this is needed to allow parsing of rdf types with whitespace in them (though it must be the last one).